src/base/sign-in-off/_base.scss

```diff
@@ -0,0 +1,8 @@
+#wb-so {
+	text-align: $wb-so-text-align;
+
+	.row {
+		background: $wb-so-background-color;
+		padding: 1em 0 0;
```

Comment:
It is needed for practically all themes, otherwise the buttons are squished on the banner and it makes it look really cramped up.